### PR TITLE
tfm-ner training module change

### PR DESCRIPTION
**Pull request by AshishMahendra**

### CODE REVIEW

File: jaseci_kit/jaseci_kit/modules/tfm_ner/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@ transformers
 pytorch
 fastapi
 uvicorn
-datasets
\ No newline at end of file
+datasets
+evaluate
+huggingface-hub
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/requirements.txt:7]: Is this required?

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tests/test_entity_extraction.py [hunk 2]
```diff
@@ -6,15 +6,13 @@
...
-    test_test_data,
-    test_entities,
     test_model_config,
     test_predict_input,
     test_predict_output,
 )
 
 
-class entity_extraction_type2_test(TestCaseHelper, TestCase):
+class TFM_NER_Test(TestCaseHelper, TestCase):
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tests/test_entity_extraction.py]: Let's name it consistently with other tests case convention `tfm_ner_test`

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tests/train.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+my name is [Shawn](name)
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tests/train.txt]: Don't think this file is actually used in the tests?

Comment (ypkang) [reply jaseci_kit/jaseci_kit/modules/tfm_ner/tests/train.txt]: remove this file.

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py [hunk 4]
```diff
@@ -36,45 +34,21 @@ def config_setup():
         model_config = json.load(jsonfile)
 
     curr_model_path = model_config["model_name"]
-    load_custom_model(curr_model_path)
+    train_data = []
+    if os.path.exists("train/train.txt"):
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py]: Change from txt to json.
Return error if `extract_entity` is called without any model loaded.
train_model should still work out of the box.

**Review by ypkang — CHANGES_REQUESTED** [review]

File: jaseci_kit/jaseci_kit/modules/tfm_ner/requirements.txt (hunk 1 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/modules/tfm_ner/requirements.txt:7]: Yes, trainer needs it

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tests/test_entity_extraction.py (hunk 2 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/modules/tfm_ner/tests/test_entity_extraction.py]: fixed

**Review by ypkang — APPROVED** [review]
LGTM

**Pull request merged**